regnety/dataset/imagenet.py

```diff
@@ -149,7 +168,58 @@ def color_jitter(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
         )
 
         return aug_images, target
-
+    
+    def _inception_style_crop(self, images, labels):
+        """
+        Applies inception style cropping
+        """
+        # # Get target metrics
+        area_ratio = tf.random.uniform((), minval=0.08, maxval=1.0)
```

Comment:
`minval` here seems very small. Is this coming from the original code?

Reply:
Yes, it is coming from the original code.